src/Core__Float.resi

```diff
@@ -1,26 +1,404 @@
+
+  /**
+  The smallest positive numeric value representable in JavaScript.
+  See [`Number.MIN_VALUE`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MIN_VALUE) on MDN.
+
+  ## Examples
+
+  ```rescript
+  Float.Constants.minValue
+  ```
+  */
+  @val
+  external minValue: float = "Number.MIN_VALUE"
+
+  /**
+  The maximum positive numeric value representable in JavaScript.
+  See [`Number.MAX_VALUE`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_VALUE) on MDN.
+
+  ## Examples
+
+  ```rescript
+  Float.Constants.minValue
+  ```
+  */
+  @val
+  external maxValue: float = "Number.MAX_VALUE"
 }
-@val external isNaN: float => bool = "isNaN"
-@val external isFinite: float => bool = "isFinite"
-@val external parseFloat: 'a => float = "parseFloat"
-@val external parseInt: 'a => float = "parseInt"
-@val external parseIntWithRadix: ('a, ~radix: int) => float = "parseInt"
-@send external toExponential: float => string = "toExponential"
-@send external toExponentialWithPrecision: (float, ~digits: int) => string = "toExponential"
-@send external toFixed: float => string = "toFixed"
-@send external toFixedWithPrecision: (float, ~digits: int) => string = "toFixed"
-@send external toPrecision: float => string = "toPrecision"
-@send external toPrecisionWithPrecision: (float, ~digits: int) => string = "toPrecision"
-@send external toString: float => string = "toString"
-@send external toStringWithRadix: (float, ~radix: int) => string = "toString"
-@send external toLocaleString: float => string = "toLocaleString"
+
+/**
+`isNaN(v)` tests if the given `v` is `NaN`.
+See [`NaN`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN) on MDN.
+
+## Examples
+
+```rescript
+Float.isNaN(3.0) // false
+Float.isNaN(Float.Constants.nan) // true
+```
+*/
+@val
+external isNaN: float => bool = "isNaN"
+
+/**
+`isFinite(v)` tests if the given `v` is finite.
+See [`isFinite`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite) on MDN.
+
+## Examples
+
+```rescript
+Float.isFinite(1.0) // true
+Float.isFinite(Float.Constants.nan) // false
+Float.isFinite(Float.Constants.positiveInfinity) // false
+```
+*/
+@val
+external isFinite: float => bool = "isFinite"
+
+/**
+`parseFloat(v)` parse the given `v` and returns a float. Leading whitespace in
+`v` is ignored. Returns `NaN` if `v` can't be parsed.
+See [`parseFloat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat) on MDN.
+
+## Examples
+
+```rescript
+Float.parseFloat("1.0") // 1.0
+Float.parseFloat("  3.14   ") // 3.14
+Float.parseFloat(3.0) // 3.0
+Float.parseFloat("3.14some non-digit characters") // 3.14
+Float.parseFloat("error")->Float.isNaN // true
+```
+*/
+@val
+external parseFloat: 'a => float = "parseFloat"
+
+/**
+`parseInt(v)` parse the given `v` and returns a float. Leading whitespace in
```

Comment:
I wonder if we should keep this + other parseInt stuff around. Doesn't make sense to have it in the Float module imo, unless I'm missing something.

Reply:
We can do this in another PR?

Reply:
Yes, for sure.